apply.sh

```diff
@@ -77,7 +77,13 @@ conf()
 
 start(){
     export PATH=${CATTLE_HOME}/bin:$PATH
-    check
+
+    if [ "${CATTLE_CHECK_NAMESERVER}" = "false" ];then
+        info Skip checking DNS nameserver
```

Comment:
This is grammatically awkward. Better phrasing:
`Skipping DNS nameserver check`